src/converters/finecoConverter.ts

```
@@ -0,0 +1,261 @@
+     */
+    public processFileContents(input: string, successCallback: any, errorCallback: any): void {
+
+        // Skip preamble lines until the actual CSV header.
+        const lines = input.split("\n");
+        const headerIndex = lines.findIndex(line => {
+            const lower = line.toLocaleLowerCase();
+            return lower.includes("operazione") && lower.includes("data valuta") && lower.includes("descrizione");
+        });
+
+        if (headerIndex === -1) {
+            return errorCallback(new Error("Could not find header row in input file!"));
```

Biome (2.4.4)

[error] The function should not return a value because its return type is void.

(lint/correctness/noVoidTypeReturn)